### PR TITLE
Delete Agent

### PR DESCRIPTION
**Pull request by jonathan-osorio**
- Added button inside the view component to delete agent 
- Added route to go back to main agent component if the method was successful

### CODE REVIEW

**Review by diogo-s4ntos — APPROVED** [review]

File: src/app/agent-management/agent-management.component.ts [hunk 1]
```diff
@@ -36,4 +37,5 @@ export class AgentManagementComponent {
     }
     )
   }
+
```
Comment (RafaelMeha) [inline src/app/agent-management/agent-management.component.ts:40]: Extra line

File: src/app/modals/delete-agent-validation-modal/delete-agent-validation-modal.component.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+@Component({
+  selector: 'app-delete-agent-validation-modal',
+  templateUrl: './delete-agent-validation-modal.component.html',
+  styleUrl: './delete-agent-validation-modal.component.css'
+})
+export class DeleteAgentValidationModalComponent {
+  agentComponent: AgentManagementComponent
+  agentId: number
+
+  constructor(public activeModal: NgbActiveModal, private modalService: NgbModal, private apiService: AgentService, private toastService: ToastService, private route: Router) { }
```
Comment (RafaelMeha) [inline src/app/modals/delete-agent-validation-modal/delete-agent-validation-modal.component.ts:18]: private modalService: NgbModal not being used

File: src/app/agent-management/agent-management.component.ts (hunk 1 above)
Comment (jonathan-osorio) [reply src/app/agent-management/agent-management.component.ts]: done :)

File: src/app/modals/delete-agent-validation-modal/delete-agent-validation-modal.component.ts (hunk 2 above)
Comment (jonathan-osorio) [reply src/app/modals/delete-agent-validation-modal/delete-agent-validation-modal.component.ts]: done

**Pull request merged**